### DISCUSSION TITLE
Improve Issue Creating /Editing

### DISCUSSION THREAD

**Discussion by il-juggler** [community/community#4845] [Mobile]
I was writing a huge isssue, and it is too easy to close the window by a accident tap outside it.

So I loose all my job for a little error, I think it could be improved by adding a confirm box when the text inside the modal/window changed.
Thanks

**Comment by fvdm** [comment]
I do get a confirmation prompt when I close the editor while creating a new issue. But not when editing.

It would be nice if it worked the same as for comments in issues and discussions. When you write a comment and leave the view, go to a different repo and come back later, the content is still remembered in the text field.

**Reply by JoshuaDeskins6** [reply]
H

**Reply by chadleap** [reply]
Agree with this being a problem. The issue editing screen on iPad leaves a lot of unused real estate outside the issue modal and any stray touch outside the borders just no-confirm closes the edit, losing everything done since last save. I've done it a lot when trying to position cursor at the beginning of a line which is very close to the edge of the window. Leads to a lot of incremental paranoia-saving
